### PR TITLE
clean arana project log output 

### PR DESCRIPTION
**Pull request by zxy-lgtm**
add :logging_config in yaml, LoggingConfig

delete : const (env) GetEnvValue

fix : model bootOptions

fixes: #650

### CODE REVIEW

**Comment by zxy-lgtm** [comment]
I don't know how to debug my code, please help

File: pkg/util/log/logging.go [hunk 1]
```diff
@@ -54,7 +53,22 @@ const (
...
+	LogPath        string `yaml:"log_path" json:"log_path"`
+	LogLevel       int    `yaml:"log_level" json:"log_level"`
+	LogMaxSize     int    `yaml:"log_max_size" json:"log_max_size"`
+	LogMaxBackups  int    `yaml:"log_max_backups" json:"log_max_backups"`
+	LogMaxAge      int    `yaml:"log_max_age" json:"log_max_age"`
+	LogCompress    bool   `yaml:"log_compress" json:"log_compress"`
+	DefaultLogName string ` yaml:"default_log_name" json:"default_log_name"`
+}
+
+type Config struct {
```
[the remark this answers is not in the document]
Comment (maronghe) [reply pkg/util/log/logging.go:67]: fake_bootstrap.yaml 文件也需要添加配置，不然单元测试会panic

File: conf/bootstrap.local-etcd.yaml [hunk 2]
```diff
@@ -30,3 +30,13 @@ config:
 supervisor:
   username: root
   password: root
+
+loggging_config:
```
Comment (maronghe) [inline conf/bootstrap.local-etcd.yaml:34]: test

**Comment by maronghe** [comment]
You can run your code using Unit Test file In `discovery_test.go` first.

**Comment by maronghe** [comment]
> You can run your code using Unit Test file In `discovery_test.go` first.

**Comment by maronghe** [comment]
> I don't know how to debug my code, please help


You can run your code using Unit Test file In discovery_test.go first.

File: pkg/boot/boot.go [hunk 3]
```diff
@@ -21,21 +21,17 @@ import (
...
-	"go.uber.org/multierr"
-)
 
-import (
 	"github.com/arana-db/arana/pkg/config"
 	"github.com/arana-db/arana/pkg/proto/rule"
 	"github.com/arana-db/arana/pkg/runtime"
 	"github.com/arana-db/arana/pkg/runtime/namespace"
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
```
Comment (jjeffcaii) [inline pkg/boot/boot.go:30]: reformat imports.

Comment (Mulavar) [reply pkg/boot/boot.go:30]: refer to imports-formatter at https://github.com/dubbogo/tools

**Review by dongzl — APPROVED** [review]

File: pkg/util/log/logging.go [hunk 4]
```diff
@@ -54,7 +53,22 @@ const (
...
+	LogMaxSize     int    `yaml:"log_max_size" json:"log_max_size"`
+	LogMaxBackups  int    `yaml:"log_max_backups" json:"log_max_backups"`
+	LogMaxAge      int    `yaml:"log_max_age" json:"log_max_age"`
+	LogCompress    bool   `yaml:"log_compress" json:"log_compress"`
+	DefaultLogName string ` yaml:"default_log_name" json:"default_log_name"`
+}
+
+//type Config struct {
+//	LoggingConfig *LoggingConfig `validate:"required,dive" yaml:"loggging_config" json:"logging_config"`
+//}
```
Comment (Mulavar) [inline pkg/util/log/logging.go:69]: please clean the useless code

File: pkg/util/log/logging.go [hunk 5]
```diff
@@ -142,57 +156,60 @@ func init() {
...
+func getValueInt(key int, defaultVal int) int {
+	if key < 0 {
 		return defaultVal
 	}
-	return value
+	return key
 }
 
-func getEnvValueBool(key string, defaultVal bool) bool {
+/*func getEnvValueBool(key string, defaultVal bool) bool {
```
Comment (Mulavar) [inline pkg/util/log/logging.go:206]: ditto

**Comment by Mulavar** [comment]
please sqush your commit and refine the commit message.

**Review by Mulavar — APPROVED** [review]

**Review by jjeffcaii — APPROVED** [review]

**Pull request merged**